src/irmin-mem/irmin_mem.ml

```diff
@@ -38,6 +38,7 @@ module Read_only (K : Irmin.Type.S) (V : Irmin.Type.S) = struct
   let v _config = Lwt.return map
 
   let close t =
+    Log.debug (fun f -> f "close");
```

Comment:
This should probably be an `info` message, but that's not important for this PR: in future, we should do a refactoring pass to make the logs consistent.

Reply:
Right.
It's okay to keep it `debug` in this PR so that we remain consistent in using `debug` everywhere.
There will be a following up of https://github.com/mirage/index/pull/105 to make the debug statements consistently useful.